packages/clay-css/src/scss/mixins/_nav.scss

```diff
@@ -29,78 +29,164 @@
 /// A mixin to create a `.nav` color variant.
 /// @param {Map} $map - A map of `key: value` pairs. The keys and value types are listed below:
 /// @example
+/// // `bg` is deprecated use `background-color` instead
```

Comment:
Since we are replacing these keys with another one, can this cause any breaking changes? could that be used by anyone?

Reply:
@matuzalemsteles I've included an alias at https://github.com/liferay/clay/blob/3fa114a3a6f4f2109054e6a08ca026627b9a0e4a/packages/clay-css/src/scss/mixins/_nav.scss#L73-L74. Both `bg` and `background-color` work, if for some reason both are used, `bg` will win over `background-color`.

Reply:
Oh I got it. So, security here is using a `setter` to ensure compatibility, this is interesting. Thanks @pat270